src/app/compressed-air-assessment/explore-opportunities/explore-opportunities-results/explore-opportunities-results.component.html

```diff
@@ -16,7 +16,15 @@
         <div class="d-flex align-items-center pl-1 pr-1 col-4">
             <span class="table-text"> &mdash; &mdash;</span>
         </div>
-        <div class="col-4">
+        <div class="col-4 d-lg-none d-flex align-items-center pl-1 pr-1">
+            <span *ngIf="(dayTypeModificationResult.allSavingsResults.savings.percentSavings) && (dayTypeModificationResult.allSavingsResults.savings.percentSavings >= 0)" class="table-text">
```

Comment:
For all the files with similar template block:
- There shouldn't be a need to group boolean operators if they're all AND / &&. Feel free to explain if there's something going on here.
- Doing (0 > whatever-val) is fine logically but 99% of the time I would express it as (whatever-val < 0)